### PR TITLE
Tag version 0.7.0

### PR DESCRIPTION
**Pull request by sriharshakandala**
## Purpose 
Tag version 0.6.1

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Review by szy21 — APPROVED** [review]

**Review by charleskawczynski — APPROVED** [review]

File: Project.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name = "Insolation"
 uuid = "e98cc03f-d57e-4e3c-b70c-8d51efe9e0d8"
 authors = ["Climate Modeling Alliance"]
-version = "0.6.0"
+version = "0.6.1"
```
Comment (simonbyrne) [inline Project.toml:4]: It looks like #51 is a breaking change? If so, this should bump the minor version.

**Review by simonbyrne — CHANGES_REQUESTED** [review]

File: Project.toml (hunk 1 above)
Comment (sriharshakandala) [reply Project.toml:4]: It removes an pre-existing default argument. So, strictly speaking, yes. I can change to `0.7.0`

**Comment by sriharshakandala** [comment]
bors try

**Comment by sriharshakandala** [comment]
bors try

**Pull request merged**